### PR TITLE
Adds error toast in login page

### PR DESCRIPTION
**Pull request by amarpathak**
Just adding error toast messages on the login page. Reference to PR: #7

### CODE REVIEW

File: pages/app/login.js [hunk 1]
```diff
@@ -11,6 +13,22 @@ const description =
...
+  const { error } = useRouter().query;
+  useEffect(() => {
+    if (error) {
+      //Pop latest if there are multiple error query in URL
+      let errorMessage;
+      if (Array.isArray(error)) {
+        errorMessage = error.pop(error);
+      } else {
+        errorMessage = error;
+      }
```
Comment (NuroDev) [inline pages/app/login.js:26]: Could probably clean this up a bit with some ternary operators 🙂 
```suggestion
      const errorMessage = Array.isArray(error) ? error.pop(error) : error;
```

File: pages/app/login.js [hunk 2]
```diff
@@ -11,6 +13,22 @@ const description =
...
+
+      toast.error(`Error: ${errorMessage}`);
```
Comment (NuroDev) [inline pages/app/login.js:28]: Can probably get rid of the `Error: ` prefix here as the error toast is already kind of visibly known as an error from the ❌  icon
```suggestion
      toast.error(errorMessage);
```

**Review by NuroDev — APPROVED** [review]

**Comment by amarpathak** [comment]
Thank you @NuroDev,  I have made the necessary changes, request you to have a look.

**Review by NuroDev — APPROVED** [review]
LGTM 👍🏻

**Review by steven-tey — APPROVED** [review]
Nice! LGTM!

CleanShot 2022-02-04 at 14 09 57@2x

**Comment by amarpathak** [comment]
Hey, @steven-tey @NuroDev  I was really excited after my first contribution and want to do more, do you have opportunities such that I can get more involved in the team and contribute more to the project?

**Pull request merged**